test/test_flake8.py

```diff
@@ -11,13 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-from ament_flake8.main import main
 import pytest
 
+from flake8.api.legacy import get_style_guide
+
 
 @pytest.mark.flake8
 @pytest.mark.linter
 def test_flake8():
-    rc = main(argv=[])
-    assert rc == 0, 'Found code style errors / warnings'
+    # Configuration options adapted from the ament_flake8
+    # https://github.com/ament/ament_lint/blob/master/ament_flake8/ament_flake8/configuration/ament_flake8.ini
+    style = get_style_guide(
+        ignore='C816,D100,D101,D102,D103,D104,D105,D106,D107,D203,D212,D404,I202',
+        import_order_style='google',
```

Comment:
Maybe this option isn't being applied and that is why the import above changes in style?

Reply:
This option relies on a flake8 plugin `flake8-import-order` which wasn't present in my virtualenv. Once the plugin was installed the import order issue above was reported correctly.

There's no apt package for the flake8 plugin which means there's not currently a way to introduce this in a way that would be bloom-able.

Reply:
Actually `flake8-import-error` should probably be listed in `tests_require` in the `setup.py` for it to work correctly in any environment.